terminal_check_aix.go

```diff
@@ -1,9 +1,9 @@
-// +build !appengine,!js,!windows,aix
+// +build aix
```

Comment:
Can this, at least, be moved to a subpackage with a `doc.go` explaining where it comes from and why it was done this way?

Reply:
I am not sure I follow, sorry. What should I move into doc.go ?

Reply:
Move all the files you just added to a `terminal` package (sub folder) and create a `doc.go` in there to add some comments.

Reply:
Right now all the functions exposed by this terminal_* group (I added only 2 files, but could've done with 1 in retrospect) are not exported (`checkIfTerminal`). If we want to keep it that way, they will need to go into "internal/terminal" instead, so even if they will become un-importable from any other projects. This change scope will grow. We can do it in a separate MR or do it here. How do you want to proceed ?

Reply:
You can create a `IsTerminal` function in the package, which would be public, just like it is in the `ssh/terminal` package. That way, no need to change any other file.

Reply:
We need to change other files to adjust imports. I moved all of them under import/terminal, please take a look

Reply:
I will add an additional review since there is more than one way to solve it

Reply:
Created https://github.com/sirupsen/logrus/pull/932 that be closer to what you have asked